src/systemc/ext/tlm_core/2/sockets/initiator_socket.hh

```diff
@@ -51,6 +51,10 @@ template <unsigned int BUSWIDTH, typename FW_IF, typename BW_IF, int N,
           sc_core::sc_port_policy POL>
 class tlm_base_target_socket;
 
+// The overloaded virtual is intended in SystemC, so we'll disable the warning.
```

Comment:
I have checked only adding this back can make two compiler happy. Would you like to change? Thanks.